packages/blocks/blocksAntd/src/blocks/ConfirmModal/ConfirmModal.js

```diff
@@ -44,12 +44,12 @@ const ConfirmModal = ({ blockId, events, content, methods, properties }) => {
         width: properties.width,
         zIndex: properties.zIndex,
         onOk: async () => {
-          await methods.triggerEvent({ name: 'onOk' });
-          methods.triggerEvent({ name: 'onClose' });
+          const response = await methods.triggerEvent({ name: 'onOk' });
+          if (response.success === false) throw response;
```

Comment:
@Gervwyk What should we be throwing here?

Reply:
No, we should not throw on unsuccessful actions.